app/src/main/java/fr/free/nrw/commons/contributions/ContributionsFragment.java

```diff
@@ -617,5 +620,20 @@ public void backButtonClicked() {
     public MediaDetailPagerFragment getMediaDetailPagerFragment() {
         return mediaDetailPagerFragment;
     }
+
+  // click listener to toggle description
```

Comment:
Would you mind adding what it means from the UI user's point of view? (tap on the explanation to hide it)
Thanks!

Reply:
You can tap on it to hide and tap again to show